modules/ROOT/pages/log-search-query-syntax.adoc

```diff
@@ -7,7 +7,7 @@ In addition to using the query builder and facet search user interface in Anypoi
 
 == Search Terms
 
-A search query consists of terms and operators. There are two types of search terms--single terms and phrases. A single term is one word, for example "test" or "hello". A phrase is a group of words surrounded by double quotes, for example "hello dolly".
+A search query consists of terms and operators. There are two types of search terms&#8212;single terms and phrases. A single term is one word, for example "test" or "hello". A phrase is a group of words surrounded by double quotes, for example "hello dolly".
```

Comment:
Can we address the user/user goals here? "To <do this>, use a search query, which consists of search terms and search operators. Search terms can be either single terms, such as "hello," or phrases, such as "hello dolly."

GLOBALLY avoid introducing sentences by using "There is..." or "There are."
Consider replacing the term and phrase with something more technical than Hello, Dolly!  ;-)